app/logConfig.py

```diff
@@ -0,0 +1,50 @@
+LOGGING_CONFIG = { 
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': { 
+        'standard_formatter': { 
+            'format': '%(asctime)s [%(levelname)s] %(name)s: %(message)s'
+        },
+        # 'request_formatter': { 
+        #     'format': '%(asctime)s [%(levelname)s] %(name)s: %(type)s : %(clientip)s : %(endpoint)s : %(message)s'
+        # }
+    },
+    'handlers': { 
+        # 'req_handler': { 
```

Comment:
Do these comments are needed?

Reply:
Not necessary. But added it so that later in time, we can have different options for loggers

Reply:
We can remove it now. The commented lines are easy to write up in future.

Reply:
Okay